Mathlib/SetTheory/PGame/Basic.lean

```diff
@@ -60,7 +60,7 @@ The material here is all drawn from
 
 An interested reader may like to formalise some of the material from
 * [Andreas Blass, *A game semantics for linear logic*][MR1167694]
-* [André Joyal, *Remarques sur la théorie des jeux à deux personnes*][joyal1997]
+* [André Joyal, *Remarques sur la théorie des jeux à deux personnes*][joyal1977]
```

Comment:
Where exactly is the difference here?

Reply:
97 vs 77

Reply:
Please call this out explicitly in the PR description, if it really was deliberate.